README.md

```diff
@@ -1,4 +1,5 @@
-Latest version is at https://github.com/jadonk/BlueDonkey
+# The Donkeys BlueDonkey Proof of Concept
+Below are instructions from the Eastern Michigan University IA Capstone project for team The Donkeys.
```

Comment:
I'm happy to give you credit in all future versions if you don't take over full ownership. :-)